app/src/main/java/org/simple/clinic/appupdate/AppUpdateHelpContact.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.appupdate
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class AppUpdateHelpContact(
+    @Json(name = "display_text")
+    val displayText: String,
+    val url: String
```

Comment:
What is the url for?

Reply:
URL will be opened when the user clicks on the contact button.
https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1644906753850759?thread_ts=1644229157.747019&cid=CFK2PHJ1L